src/Speedrun.cpp

```diff
@@ -30,14 +30,21 @@ void Speedrun::on_draw_ui() {
     locked->draw("Lock Window");
     in_game_time->draw("In Game Time");
     game_rank->draw("Game Rank");
+
+    if (game_rank->value())
```

Comment:
Of course, done!

Btw since that PR some DA strats were found (e.g. for Del Lago 7 hits on Pro), idk if it's still relevant to have a show/hide options for the points since they're very important.

Reply:
It can't hurt to have options right?